From 5475b9227bfc611336f3ec5e872c2ecdc279189c Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Sat, 19 Jul 2014 04:26:24 -0400
Subject: [drm] i915: Rename gtt_bus_addr to gtt_phys_addr

Message-id: <20140719042624.19438.39790.stgit@gir.stowe>
Patchwork-id: 85997
O-Subject: [RHEL7.1 PATCH 35/81] drm/i915: Rename gtt_bus_addr to gtt_phys_addr
Bugzilla: 1110895
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110895
Upstream Status: 21c346075c0694581303ff04ff2be021587e4b40

commit 21c346075c0694581303ff04ff2be021587e4b40
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Sat Dec 21 10:52:52 2013 -0700

    drm/i915: Rename gtt_bus_addr to gtt_phys_addr

    We're dealing with CPU physical addresses here, which may be different from
    bus addresses, so rename gtt_bus_addr to gtt_phys_addr to avoid confusion.

    No functional change.

    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
    Reviewed-by: Daniel Vetter <daniel.vetter@ffwll.ch>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/gpu/drm/i915/i915_gem_gtt.c b/drivers/gpu/drm/i915/i915_gem_gtt.c
index 1f7b4ca..1d26ac6 100644
--- a/drivers/gpu/drm/i915/i915_gem_gtt.c
+++ b/drivers/gpu/drm/i915/i915_gem_gtt.c
@@ -881,7 +881,7 @@ static int gen6_gmch_probe(struct drm_device *dev,
       unsigned long *mappable_end)
 {
  struct drm_i915_private *dev_priv = dev->dev_private;
- phys_addr_t gtt_bus_addr;
+ phys_addr_t gtt_phys_addr;
  unsigned int gtt_size;
  u16 snb_gmch_ctl;
  int ret;
@@ -907,10 +907,10 @@ static int gen6_gmch_probe(struct drm_device *dev,
  *gtt_total = (gtt_size / sizeof(gen6_gtt_pte_t)) << PAGE_SHIFT;
 
  /* For Modern GENs the PTEs and register space are split in the BAR */
- gtt_bus_addr = pci_resource_start(dev->pdev, 0) +
+ gtt_phys_addr = pci_resource_start(dev->pdev, 0) +
   (pci_resource_len(dev->pdev, 0) / 2);
 
- dev_priv->gtt.gsm = ioremap_wc(gtt_bus_addr, gtt_size);
+ dev_priv->gtt.gsm = ioremap_wc(gtt_phys_addr, gtt_size);
  if (!dev_priv->gtt.gsm) {
   DRM_ERROR("Failed to map the gtt page table\n");
   return -ENOMEM;
-- 
1.7.1